metricscheck.log

```diff
@@ -0,0 +1,27 @@
+{"level":"info","ts":1579123328.0971992,"caller":"metricswatcher/metrics.go:75","msg":"Request duration = 0.00097710"}
```

Comment:
I think this was added by error.

Reply:
Ops, you're right. Misuse of `git add -A` after I updated `.gitignore`.